### ISSUE TITLE
[Discussion] Scheduler should give 7 days schedule but currently its returning only one schedule of that current day

### ISSUE THREAD

**Issue by Tushar-4781** [tijlleenders/ZinZen-scheduler#216]
My Time page has to show the schedule for all 7 days. But despite of passing startDate and endDate it is returning only one schedule. shouldn't it return 7 schedules of the respective day?
![image](https://user-images.githubusercontent.com/39093015/210299474-8b9519ee-c970-4870-a833-4cde4c5e0978.png)

**Comment by tijlleenders** [comment]
The current assumption is that frontend would get two collections: 
- scheduled tasks
- impossible tasks

Dividing them into seven days is a frontend concern currently.

@Tushar-4781 Would you like the scheduler to do the dividing?  
That would make sense from performance perspective.
If so, do you want:
- a seven day collection with impossible/scheduled per day
- two collections (impossible/scheduled) with subdivision per day for each
?

Also - do you want empty collections if there is nothing to report?

**Comment by Tushar-4781** [comment]
@tijlleenders yes, I expect 7 days schedule in the same format. each day having two collections:
- scheduled tasks
- impossible tasks

Expected Output 
 { 
   dayName :  
         { 
              scheduledTasks: [ ], 
              impossibleTasks: [ ] 
         } 
 } [ ]

**Comment by egithinji** [comment]
On the front end, what problem is being caused by the current way the scheduler is outputting the tasks? i.e. two collections: scheduled tasks & impossible tasks with the tasks having start and deadlines?

**Comment by tijlleenders** [comment]
This discussion has been resolved.
#742 will be implemented using #889
